kvmagent/kvmagent/test/vmmeta_testsuite/local/test_file_prefix_rebase.py

```
@@ -0,0 +1,295 @@
+import os
+import shutil
+import tempfile
+from unittest import TestCase
+
+from kvmagent.test.utils import pytest_utils
+from kvmagent.test.utils.stub import *
```

Ruff (0.15.7)

[error] `from kvmagent.test.utils.stub import *` used; unable to detect undefined names

(F403)